person.rb

```diff
@@ -0,0 +1,32 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getters
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  # setters
+
+  # -------------------
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
`of_age?` should be a private method as per the project requirements. Kindly make this method private.